apps/vscode/editor/src/Links.tsx

```diff
@@ -3,43 +3,42 @@ import { openUrl } from './utils/url'
 
 export function Links() {
 	return (
-		<TldrawUiMenuGroup id="links">
```

Comment:
Should probably pull this out somewhere, but we don't have a really good spot for it 🤷‍♂️